helix-core/src/test/java/org/apache/helix/integration/rebalancer/DelayedAutoRebalancer/TestDelayedAutoRebalance.java

```diff
@@ -49,7 +49,7 @@
   protected static final int PARTITIONS = 5;
   // TODO: remove this wait time once we have a better way to determine if the rebalance has been
   // TODO: done as a reaction of the test operations.
-  protected static final int DEFAULT_REBALANCE_PROCESSING_WAIT_TIME = 1000;
+  protected static final int DEFAULT_REBALANCE_PROCESSING_WAIT_TIME = 3000;
```

Comment:
Are we sure this is causing the trouble?
Based on what I tested before, 1 second is already a very long time. If with 1 second, the test is still not stable, then it might be other problems.

Let me be clear,
1. Changing this time just prolong the test, not a big deal.
2. I'm more concerned that this won't really make the tests more stable.